File: ClientApp/src/app/models/PersonDto.ts

import { DTOBaseClass } from "./DTOBaseClass";
  
    
import {AddressDto} from "./AddressDto";

import {DogDto} from "./DogDto";



 
export class PersonDto extends DTOBaseClass{ 


public static AddChildProperty_PersonDto_Addresses : string= "addresses";

public static AddChildProperty_PersonDto_Dogs : string= "dogs";



id: string;  

firstname: string;  

lastname: string;  

addresses: AddressDto[];  

dogs: DogDto[];  

}

File: DTO/PersonDto.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace normalizrTestProject.DbModels.Dto
{
    public class PersonDto
    {
        public Guid Id { get; set; }

        public string Firstname { get; set; }

        public string Lastname { get; set; }

        public List<AddressDto> Addresses { get; set; }

        public List<DogDto> Dogs { get; set; }
    }
}